File: src/net/pryoscode/CustomDiscordRPC/menu/items/GitHub.java
package net.pryoscode.CustomDiscordRPC.menu.items;

import net.pryoscode.CustomDiscordRPC.Main;
import java.awt.*;
import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;

public class GitHub extends MenuItem {

    public GitHub() {
        setLabel("GitHub");
        addActionListener(new ActionListener() {
            @Override
            public void actionPerformed(ActionEvent e) {
                Main.openURL("https://github.com/PryosCode/CustomDiscordRPC");
            }
        });
    }

}

File: src/main/kotlin/net/pryoscode/customdiscordrpc/menu/items/GitHub.kt
package net.pryoscode.customdiscordrpc

import java.awt.MenuItem
import java.awt.event.ActionListener
import java.awt.event.ActionEvent
import java.awt.Desktop
import java.net.URL

class GitHub : MenuItem, ActionListener {

    constructor() {
        setLabel("GitHub")
        addActionListener(this)
    }

    override fun actionPerformed(event: ActionEvent) {
        Desktop.getDesktop().browse(URL("https://github.com/PryosCode/CustomDiscordRPC").toURI())
    }

}